src/gleam/erlang/file.gleam

```diff
@@ -111,32 +111,146 @@ pub type Reason {
   NotUtf8
 }
 
+/// TODO
+///
+pub type FileType {
+  Device
+  Directory
+  Other
+  Regular
+  Symlink
+}
+
+/// TODO
+///
+pub type Access {
+  NoAccess
+  Read
+  ReadWrite
+  Write
+}
+
+/// TODO
+///
+pub type FileInfo {
+  FileInfo(
+    size: Int,
+    file_type: FileType,
+    access: Access,
+    atime: Int,
+    mtime: Int,
+    ctime: Int,
```

Comment:
Do you think it'd be worth spelling these out in full? We tend to avoid abbreviations

Reply:
I'd tend to agree; however, we have at least one issue here.
As per https://www.erlang.org/doc/man/file.html#read_file_info-1

## `atime`

The last time the file was read.

## `mtime`

The last time the file was written.

## `ctime`

The interpretation of this time field depends on the operating system. On Unix, it is the last time the file or the inode was changed. In Windows, it is the create time.

## Fullnames
- `access_time` or `accessed_time`
- `modify_time` or `modified_time` or `modification_time`
- `???_time` (possibly `meta_or_create_time` or `change_or_create_time` or the Gleam-like `crange_time` — which also has the same number of letters as `access_time` and `modify_time` :stuck_out_tongue_winking_eye:)

Perhaps `atime`, `mtime`, and `ctime` are familiar enough and clearer as is.

Reply:
Fair enough, short names it is!